fixtures/stairville/stagepar-cx-2.json

```
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "StagePar CX-2",
+  "categories": ["Color Changer", "Dimmer", "Effect"],
+  "meta": {
```

A comment about the LEDs should be here so people know about the interesting LED layout for this fixture
```suggestion
  "comment": "The outer ring of LEDs are RGBA and an inner ring LEDs are RGBW",
  "meta": {
```